include/tins/sniffer.h

```diff
@@ -449,8 +449,14 @@ HandlerProxy<T> make_sniffer_handler(T* ptr,
 /**
  * \brief Iterates over packets sniffed by a BaseSniffer.
  */
-class SnifferIterator : public std::iterator<std::forward_iterator_tag, Packet> {
+class SnifferIterator {
 public:
+    typedef std::forward_iterator_tag iterator_category;
```

Comment:
Maybe replace typedef with using

Reply:
OK, done

Reply:
Unfortunately the default build mode still supports C++03 so we can't use `using` here and have to stick to typedef. It's probably about time to drop support for C++03 but we can't just yet.